packages/taquito-core/package.json

```diff
@@ -59,5 +59,11 @@
     "collectCoverageFrom": [
       "src/**/*.{js,ts}"
     ]
+  },
+  "dependencies": {
+    "json-stringify-safe": "^5.0.1"
```

Comment:
is this dependency necessary?

Reply:
See the response to https://github.com/ecadlabs/taquito/pull/2657#discussion_r1338715294